pipeline_dp/accumulator.py

```diff
@@ -29,10 +29,21 @@ def create_accumulator_params(
 ) -> typing.List[AccumulatorParams]:
     accumulator_params = []
     if pipeline_dp.Metrics.COUNT in aggregation_params.metrics:
-        # TODO: populate CountParams from budget_accountant when it is ready
+        request_budget = budget_accountant.request_budget
+        low, high, max_partitions_contributed,\
+        max_contributions_per_partition = \
+            aggregation_params.low, aggregation_params.high, \
+            aggregation_params.max_partitions_contributed, \
+            aggregation_params.max_contributions_per_partition
+        noise_kind = NoiseKind.GAUSSIAN
```

Comment:
Ah, I missed to add noise_kind to aggregation_params. Could you please add `noise_kind` to aggregation_params ?

Reply:
noise_kind was added to aggregate_params on the PR merged today